msa/black-box-tests/src/test/java/org/thingsboard/server/msa/ui/base/AbstractDriverBaseTest.java

```diff
@@ -67,7 +66,7 @@ abstract public class AbstractDriverBaseTest extends AbstractContainerTest {
 
     @SneakyThrows
     @BeforeMethod
-    public void openBrowser() {
+    public void startUp() {
         log.info("===>>> Setup driver");
         testRestClient.login(TENANT_EMAIL, TENANT_PASSWORD);
```

Comment:
Why we login every test method?
 Please, try to log before class or before suite (up to your needs). There is plenty of possibilities:
 
![image](https://user-images.githubusercontent.com/79898499/222769202-1868f718-a76e-41b4-984c-13d8a9ada794.png)